docker-compose.yml

```diff
@@ -1,40 +1,44 @@
-version: "3.9"
+version: "3.7"
```

Comment:
why downgrade ?

Reply:
To use the oldest possible version. The `init` argument in backend needs version 3.7, so that's what I've gone for.

Reply:
Do you know why they removed it?